hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CommonConfigurationKeys.java

```diff
@@ -261,6 +261,11 @@ public class CommonConfigurationKeys extends CommonConfigurationKeysPublic {
   public static final int
       HADOOP_SECURITY_DNS_LOG_SLOW_LOOKUPS_THRESHOLD_MS_DEFAULT = 1000;
 
+  /**
+   * Script used to get server principal when _REALM pattern is used
```

Comment:
Could you add a small description about the input of the script in the commend (either an env variable or a positional argument, see another comment about this)

Reply:
done